backend/lib/tasks/api.rake

```diff
@@ -0,0 +1,85 @@
+namespace :api do
```

Comment:
not the most descriptive of names - should you be telling this rake task fetches restaurants?